### ISSUE TITLE
Public transport routes input as .add or .rou

### ISSUE THREAD

**Issue by ArmelliniMG** [eclipse/sumo#8699]
If our recommendation is to define the file with the public transport routes as an additional file, we should modify the documentation, bacause this is not so clear.
The use of public transport routes as an additional file is to enable person rides with public transport. Is this then already defined? Should we change the documentation and adopt a global format?

If so, some docs to modify:
https://sumo.dlr.de/docs/Tutorials/PublicTransport.html#sumocfg
https://sumo.dlr.de/docs/Simulation/Public_Transport.html
https://sumo.dlr.de/docs/Specification/Persons.html#rides
https://sumo.dlr.de/docs/IntermodalRouting.html#defining_public_transport

**Comment by namdre** [comment]
Yes, to adding the recomendation of loading the pt routes with **--additional-files**.
I'm not so sure whether we should really recommend a different file extension.I haven't got a strong gut feeling either way (@behrisch ?)

**Comment by behrisch** [comment]
I think the file suffix is a good indication for the user so yes let's change it.

**Comment by behrisch** [comment]
Needs documentation, ptlines2flows should generate .add.xml files and duarouter should load routes from additionals
